maps/stellardelight/stellar_delight_defines.dm

```diff
@@ -0,0 +1,699 @@
+//Normal map defs
```

Comment:
It would be nice if, in the future at some point, this was as compartmentalized into different files as tether was with turfs and objects and shuttles whatnot in different files. Not urgent though.